tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_sync_committee_updates.py

```diff
@@ -41,6 +41,15 @@ def run_sync_committees_progress_test(spec, state):
     # of this `EPOCHS_PER_SYNC_COMMITTEE_PERIOD`
     third_sync_committee = spec.get_next_sync_committee(state)
 
+    # Ensure assignments have changed:
+    assert state.next_sync_committee != second_sync_committee
+    if current_period > 0:
+        assert state.current_sync_committee != first_sync_committee
+    else:
+        # Current and next are duplicated in genesis period so remain stable
+        assert state.current_sync_committee == first_sync_committee
+
+    # Ensure expected committees were calculated
     assert state.current_sync_committee == second_sync_committee
     assert state.next_sync_committee == third_sync_committee
 
```

Comment:
[L24, L25](https://github.com/ethereum/eth2.0-specs/pull/2502/files#diff-21fc87e491da53e8990aa2cc3ab52b4ed3f1576dd74406b24ad079acb3b0739fR24-R25):
```python
    first_sync_committee = state.current_sync_committee
    second_sync_committee = state.next_sync_committee
```

In this case, sync committees were assigned instead of being *mutated* during `run_epoch_processing_with`. So that the `first_sync_committee`/`second_sync_committee` checks were fine.

However, I think it would be safer regarding testing logic:
```python
    first_sync_committee = state.current_sync_committee.copy()
    second_sync_committee = state.next_sync_committee.copy()
```

Reply:
yeah, copy is safer to any weird changes under the hood 👍